server/routes/serviceProviderReferrals/dashboardPresenter.test.ts

```diff
@@ -3,21 +3,6 @@ import serviceCategoryFactory from '../../../testutils/factories/serviceCategory
 import sentReferralFactory from '../../../testutils/factories/sentReferral'
 
 describe(DashboardPresenter, () => {
-  describe('tableHeadings', () => {
```

Comment:
this test doesn't really appear to be testing anything so instead of copying the new code from the presenter in here i thought best to just remove